packages/framework/src/handler.ts

```diff
@@ -275,17 +275,17 @@ export class NovuRequestHandler<Input extends any[] = any[], Output = any> {
          * Log bridge server errors to assist the Developer in debugging errors with their integration.
          * This path is reached when the Bridge application throws an error, ensuring they can see the error in their logs.
          */
-        // eslint-disable-next-line no-console
-        console.error(error);
+        // eslint-disable-next-line @typescript-eslint/no-base-to-string
+        log((l) => l.error(error.message || error.toString()));
       }
 
       return this.createError(error);
     } else if (isPlatformError(error)) {
       return this.createError(error);
     } else {
       const bridgeError = new BridgeError(error);
-      // eslint-disable-next-line no-console
-      console.error(bridgeError);
+      // eslint-disable-next-line @typescript-eslint/no-base-to-string
+      log((l) => l.error(bridgeError.message || bridgeError.toString()));
```

Comment:
feels a bit complex and not the usual format of logging. what are the cons of doing something like this?


```suggestion
      log.error(bridgeError.message || bridgeError.toString());
```



in case of 
```
    log((l) => `${prefix} ${l.emoji.STEP} Discovered stepId: '${step.stepId}'\tType: '${step.type}'`);
```
we could do 
```
    log.info(`${prefix} ${log.emoji.STEP} Discovered stepId: '${step.stepId}'\tType: '${step.type}'`);
```

the formatting should be the responsibility of the logger user; this way, it is easier to read. in addition, having the default log level hidden is a bit confusing and not straightforward.